cmake/options.cmake

```diff
@@ -200,6 +200,10 @@ macro(drake_setup_options)
     DEPENDS "HAVE_BOT_CORE_LCMTYPES"
     "robotlocomotion LCM types")
 
+  drake_system_dependency(VTK REQUIRES VTK VERSION 5.8
```

Comment:
dang, I had this comment "pending" and didn't realize i needed to press a button to post it.  I think this will be broken on mac?